eBookReader/chat.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Q&A</title>
+	<link rel="stylesheet" type="text/css" href="styleIndex.css"/>
+	<link rel="stylesheet" type="text/css" href="debugStyle.css"/>
+</head>
+<body>
+<header>
+	<p class = "name">John Doe</p>
+	<p class = "userStats">
+		Contribution points: 91
+		<a href="#" class="iconButton">?</a>
+	</p>
+</header>
+
+<h2>According to the author, why do mass extinctions happen?</h2>
+
+<div class = "chatPanel">
```

Comment:
In terms of coding style conventions, we will adopt attribute="..." (no space between class, equal sign, and quotes) for attributes, rather than attribute = "..." (spaces between each). May seem like a small thing, but it pays off to be consistent. Please update all attributes in this page accordingly.